examples/kafka-authentication-sasl-plain-consumer/kafka_authentication_sasl_plain_consumer.bal

```diff
@@ -9,14 +9,13 @@ kafka:AuthenticationConfiguration authConfig = {
     // Check Ballerina `config` APIs to see how to use encrypted values instead of plain text values here.
     username: "ballerina",
     password: "ballerina-secret"
-
 };
 
 kafka:ConsumerConfiguration consumerConfig = {
-    groupId:"test-group",
+    groupId: "test-group",
     clientId: "sasl-consumer",
-    offsetReset:"earliest",
-    topics:["topic-sasl"],
+    offsetReset: "earliest",
+    topics: ["topic-sasl"],
     // Provide the relevant authentication configuration record to authenticate the consumer.
```

Comment:
```suggestion
    // Provides the relevant authentication configuration record to authenticate the consumer.
```
Should be `provide` if this is an instruction for the user.

Reply:
Kept it as `provide` since its an instruction.